charts/cluster-addons/values.yaml

```diff
@@ -117,6 +117,63 @@ openstack:
       volumeType:
       # The allowed topologies for the storage class
       allowedTopologies:
+  csiManila:
+    # Indicates if the Manila CSI should be enabled
+    # By default, it is disabled if the OpenStack integrations are enabled
+    # See https://github.com/kubernetes/cloud-provider-openstack/blob/master/charts/manila-csi-plugin/values.yaml
+    enabled: false
+    cephfs:
+      enabled: true
+    nfs:
+      enabled: true
+    chart:
+      repo: https://kubernetes.github.io/cloud-provider-openstack
+      name: openstack-manila-csi
+      version: 2.28.0
+    values: {}
+    # Region is needed for auth
+    region: RegionOne
+    # Variables affecting the definition of the storage classes
+    cephfsStorageClass:
+      # Indicates if the storage class should be enabled
+      enabled: true
+      # The name of the storage class
+      name: csi-manila-cephfs
+      # Which mounting method to use? kernel or fuse
+      cephfsMounter: kernel
+      # The manila share type
+      type: cephfs
+      # Indicates if the storage class should be annotated as the default storage class
+      isDefault: false
+      # The reclaim policy for the storage class
+      reclaimPolicy: Delete
+      # Indicates if volume expansion is allowed
+      allowVolumeExpansion: true
+      # The Cinder availability zone to use for volumes provisioned by the storage class
+      manilaAvailabilityZone:
+      #
+      autoTopology:
+      # The allowed topologies for the storage class
+      allowedTopologies:
+    nfsStorageClass:
+      # Indicates if the storage class should be enabled
+      enabled: false
```

Comment:
This should be `true` but the storage class gated on `.Values.openstack.csiManila.enabled` and `.Values.csi.nfs.enabled`.